rust/agama-dbus-server/src/questions.rs

```diff
@@ -58,22 +69,19 @@ impl LuksQuestionObject {
     pub fn set_password(&mut self, value: &str) -> () {
         self.0.password = value.to_string();
     }
-
-    #[dbus_interface(property)]
-    pub fn attempt(&self) -> u8 {
-        self.0.attempt
-    }
 }
 
 /// Question types used to be able to properly remove object from dbus
 enum QuestionType {
-    Generic,
-    Luks,
+    Base,
+    BaseWithPassword,
 }
 
 trait AnswerStrategy {
-    /// TODO: find way to be able to answer any type of question, not just generic
+    /// Provides answer for generic question
```

Comment:
```suggestion
    /// Provides answer for generic question.
    /// Returns one of the question options.
```

What does the `None` return value mean?

Reply:
`None` means in both cases that Answering strategy does not provide it. As FileStrategy will be another implementation of this trait and if answer is not specified there, it should return None.